samples/react-sp-bookmarks/README.md

```diff
@@ -0,0 +1,120 @@
+    - `gulp bundle --ship`
+    - `gulp package-solution --ship`
+2. Install into your SharePoint app catalog and add it to a SharePoint site.
+3. Navigate to your site, eg. **https://\<tenant>.sharepoint.com/sites/\<your site>**
+4. Create a custom list with Title and Description(internal name) of type Enhanced rich text. 
+5. Populate the list with some items
+6. Navigate to a page on your site where the custom list is created
+7. Add the Web Part to the page and configure it.
+
+## Features
+
+1. **Bookmark Management**:
+   - Add new bookmarks with a title, URL, and icon.
+   - The webpart supports FluentUI icon names (https://developer.microsoft.com/en-us/fluentui#/styles/web/icons)
+   - Edit existing bookmarks to update their details.
+   - Delete bookmarks from the list.
+
+2. **Grid Layout**:
+   - Displays bookmarks in a responsive grid layout.
+   - Each bookmark includes a title, icon, and link.
+
+3. **Theming**:
+   - Supports **light** and **dark** themes for better user experience.
+   - Automatically adapts to the SharePoint site's theme.
+
+4. **SharePoint Integration**:
+   - Stores bookmarks in a SharePoint list.
+   - Uses PnP JS for seamless interaction with SharePoint APIs.
+
+5. **User-Friendly UI**:
+   - Intuitive interface with a modal dialog for adding/editing bookmarks.
+
+
+## Help
+
+We do not support samples, but we this community is always willing to help, and we want to improve these samples. We use GitHub to track issues, which makes it easy for  community members to volunteer their time and help resolve issues.
+
+If you're having issues building the solution, please run [spfx doctor](https://pnp.github.io/cli-microsoft365/cmd/spfx/spfx-doctor/) from within the solution folder to diagnose incompatibility issues with your environment.
+
+If you encounter any issues while using this sample, [create a new issue](https://github.com/pnp/sp-dev-fx-webparts/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Abug-suspected%2Csample%3A%20REACT-SP-BOOKMARKS&template=bug-report.yml&sample=REACT-SP-BOOKMARKS&authors=@SudeepGhatakh&title=REACT-SP-BOOKMARKS).
+
+For questions regarding this sample, [create a new question](https://github.com/pnp/sp-dev-fx-webparts/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Aquestion%2Csample%3A%20REACT-SP-BOOKMARKS&template=bug-report.yml&sample=REACT-SP-BOOKMARKS&authors=@SudeepGhatakh&title=REACT-SP-BOOKMARKS).
+
+Finally, if you have an idea for improvement, [make a suggestion](https://github.com/pnp/sp-dev-fx-webparts/issues/new?assignees=&labels=Needs%3A+Triage+%3Amag%3A%2Ctype%3Aenhancement%2Csample%3A%20REACT-SP-BOOKMARKS&template=bug-report.yml&sample=REACT-SP-BOOKMARKS&authors=@SudeepGhatakh&title=REACT-SP-BOOKMARKS).
+
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+
```

Comment:
the image at the end of the readme template is also needed here. We call it a [visitor stats image](https://github.com/pnp/sp-dev-fx-webparts/blob/main/CONTRIBUTING.md#visitor-stats-image) and you may read about it in the contribution guide. It is needed for telemetry reasons. Lets add it